elastic/beats
# [auditbeat] fim: implement ebpf backend

## Proposed commit message

This PR adds an additional opt-in eBPF backend to the file_integrity module. See *related issues* for more context.

## Checklist


## Author's Checklist

- [x] open source https://github.com/elastic/ebpfevents
- [ ] support setxattrs file modification events
- [ ] support directories creation events(?)

## How to test this PR locally

Enable the eBPF backend in the file_integrity module config by specifying `force_backend: ebpf` and observe file events after running auditbeat.

## Related issues

- https://github.com/elastic/integrations/issues/7401
- #37796
